e2e/screenshot.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+import * as gulp from 'gulp';
+import * as path from 'path';
+import {browser} from 'protractor';
+
+
+function initializeEnvironment(jasmine: any) {
+  var reporter = new jasmine.JsApiReporter({});
+  reporter.specStarted = function(result: any) {
+    jasmine.getEnv().currentSpec = result.fullName;
```

Comment:
Instead of storing this to `jasmine.getEnv()`, would it work to create a variable at the file level and update that? E.g.,
```ts
let currentJasmineSpecName = '';

function initializeEnvironment(jasmine: any) {
  jasmine.getEnv().addReporter(new jasmine.JsApiReporter({
    specStarted: (result: any) => {
      currentJasmineSpecName = result.fullName;
    }
  }));
}
```